### PR TITLE
Marques johnson

### PR DESCRIPTION
**Pull request by MarquesJ8023**

### CODE REVIEW

File: js/index.js [hunk 1]
```diff
@@ -1 +1,93 @@
 // Your code goes here
+
+
+
+const titleH1 = document.querySelector("h1");
+
+window.addEventListener("wheel", e => {
+  if (e.wheelDelta === -120) {
+    titleH1.textContent = "Young";
```
Comment (ebcitron) [inline js/index.js:9]: Tryna take it easy?

File: js/index.js [hunk 2]
```diff
@@ -1 +1,93 @@
...
+  } else {
+    titleH1.textContent = "Jeezy";
```
Comment (ebcitron) [inline js/index.js:11]: and so

File: js/index.js [hunk 3]
```diff
@@ -1 +1,93 @@
...
+  }
+});
+
+//Nav items random colors
+
+const navItems = document.querySelectorAll("a");
+navItems.forEach(item => {
+  item.addEventListener("mousemove", _ => {
```
Comment (ebcitron) [inline js/index.js:19]: This is really cool, Good work man.

**Review by ebcitron — APPROVED** [review]
Great job  man, Looks distinct

**Comment by MarquesJ8023** [comment]
Lol... Thanks. Trying to make it as easy as possible

**Pull request merged**